include/mbedtls/mbedtls_config.h

```diff
@@ -2552,7 +2552,7 @@
  *
  * \note See also our Knowledge Base article about porting to a new
  * environment:
- * https://tls.mbed.org/kb/how-to/how-do-i-port-mbed-tls-to-a-new-environment-OS
+ * https://mbed-tls.readthedocs.io/en/latest/kb/how-to/how-do-i-port-mbed-tls-to-a-new-environment-OS
```

Comment:
No trailing `/` on any of these (but they've been added in several of the changes further up this PR)